deepdrivemd/models/keras_cvae_stream/model.py

```diff
@@ -0,0 +1,395 @@
+"""
```

Comment:
We should only have a single copy of the keras CVAE code in the repository. Let's do a diff check between the streaming/non-streaming implementations and make any updates necessary to support both. I think I turned off the ModelCheckpoint mechanism in the non-streaming version, but we can make this optional.

Reply:
Done. You might want to check that I did not break your code.